FalmouthFalcons/iteration-2
# Delete customers teardown method added

Description:
_Fridays code didn't properly push._
Complete all requirements for ticket 2. Showing created customers and active customers.

Number of Fixes:
Ticket 2

Related Ticket(s):
Ticket 2

Problem to Solve:
None

Proposed Changes:
Complete all requirements for ticket 2. Showing created customers and active customers.

Expected Behavior:
Showing created customers and active customers.

Steps to Test Solution

Run main.rb to add customer
Run main.rb to show active customer
Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

Deploy Notes:

Notes regarding deployment the contained body of work.
These should note any db migrations, etc.
